specs/commandline/policy.md

```diff
@@ -67,6 +67,110 @@ The `notation policy` command provides a user-friendly way to manage trust polic
 }
 ```
 
+Policy language version 1.1 added support for verifying signatures associated with blob artifacts. User can use `scopes` field as a Policy selector string to decide which Policy gets applied to which blob. The `--policy-scope` argument provided in `notation blob verify` command will dictate which Policy gets picked from the policy configuration file and applied for verification. To get started with verifying blob signatures, users can refer to the following trust policy configuration sample. In this sample, there are three policies configured for different requirements:
+
+- The Policy named "blob-verification-policy" is for verifying blob artifacts signed by Wabbit Networks and scoped to `blob-verification-selector`.
+- Policy named "skip-blob-verification-policy" is for skipping verification on blob artifacts scoped to `skip-blob-verification-selector`.
+- Policy "wildcard-blob-verification-policy" is for auditing verification results when user wants to apply a wildcard policy by not providing `--policy-scope` argument in `notation blob verify` command.
+
+```jsonc
+{
+    "version": "1.1",
+    "trustPolicies": [
+        {
+            "name": "blob-verification-policy",
+            "scopes": [ 
+              "blob:blob-verification-selector"
+            ],
+            "signatureVerification": {
+                "level": "strict"
+            },
+            "trustStores": [ 
+              "ca:wabbit-networks",
+            ],
+            "trustedIdentities": [
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Security Tools"
+            ]
+        },
+        {
+            "name": "skip-blob-verification-policy",
+            "scopes": [ "blob:skip-blob-verification-selector" ],
+            "signatureVerification": {
+              "level" : "skip" 
+            }
+        },
+        {
+            "name": "wildcard-blob-verification-policy",
+            "scopes": [ "blob:*" ],
+            "signatureVerification": {
+              "level" : "audit"
+            },
+            "trustStores": ["ca:acme-rockets"],
+            "trustedIdentities": ["*"]
+        }
+    ]
+}
+```
+
+Note: Policy language version 1.1 renamed the field `registryScopes` from version 1.0 to `scopes`. The new field accepts values with prefixes `oci` or `blob` to limit a scope value to either OCI signature verification or Blob signature verification. While scope values with `blob` prefix can be of free-form text, values with `oci` prefix must be valid OCI references. `notation` supports both policy language versions 1.0 and 1.1. However, `notation` rejects policy configuration files with mixed terminology i.e. both `registryScopes` and `scopes` defined in a single configuration file. Users migrating from 1.0 to 1.1 can simply rename `registryScopes` to `scopes` and prefix the values with `oci`.
+Below is a sample Policy configuration file that verifies OCI artifacts using `scopes` field.
```

Comment:
Maybe a policy file including policies for both OCI and blob is the common case?

```
{
    "version": "1.1",
    "trustPolicies": [
        {
            "name": "wabbit-networks-images",
            "scopes": [ 
              "oci:registry.acme-rockets.io/software/net-monitor",
            ],…
         },
         {
           "name": "global-policy-for-OCI",
            "scopes": [ 
              "oci:*",
            ],…
         },
        {
            "name": "blob-policy",
            "scopes": [ 
              "blob:blob-verification-selector",
            ],…
         },
         {
           "name": "global-policy-for-blob",
            "scopes": [ 
              "blob:*",
            ],…
         },
```

Reply:
Added few blob policies